dir.props

```diff
@@ -34,9 +34,9 @@
   <Import Project="build/OutputDirectories.props" />
   <Import Project="build/BuildDefaults.props" />
   <Import Project="build/Stage0.props" />
-  <Import Project="build/CrossGen.props" />
   <Import Project="build/VersionBadge.props" />
   <Import Project="build/BundledRuntimes.props" />
+  <Import Project="build/CrossGen.props" />
```

Comment:
This now has a dependency on properties that are set in BundledRuntimes, it had to move down.